platform/commonUI/edit/res/templates/create/locator.html

```diff
@@ -26,4 +26,22 @@
                             ng-model="treeModel">
         </mct-representation>
     </div>
+    
+    <!-- Create New Folder Action -->
+    <div class="newFolderCreation" style="margin-top:10px; position:absolute; left:0px;">
+        
+        <!-- New folder button, triggers create new folder action. -->
+        <div ng-show="!newFolderCreationTriggered">
```

Comment:
Is there any specific reason for using ng-show instead of ng-if here?
I think you want it in the DOM if `newFolderCreationTriggered` is false and vice-versa
What is your suggestion of using ng-if here?